### DISCUSSION TITLE
Audio stuttering on OSMC Kodi Vero 4K+

### DISCUSSION THREAD

**Discussion by kocicky** [CastagnaIT/plugin.video.netflix#1388] [Q&A] [answered]
I have an issue with audio playback. On some movies/tv series with dramatic music (now Umbrella Academy), sound is stuttering. It did to me before but I solved it by changing Widevine version. No I changed Widevine through SlyGuy plugin back to 4.10.2252.5 with no change (I also tried 4.10.2252.0). I tried to change the resolution from 720p to 1080p and even to 576p but with no luck/ no change. Same issue is with HBO (Game of Thrones). Few months ago it worked fine.

Any suggestions how to debug it/solve it?

**Comment by CastagnaIT** [comment, marked as answer]
Widevine do not have influence to the audio then change Widevine version has no sense
audio is directly handled by Kodi
try to see if "Sync playback to display" is enabled in Kodi video settings, if so disable it
and try check if you have changed some other kodi audio settings

**Reply by kocicky** [reply]
I tried it, "Sync playback to display" is disabled. I tried another options with no luck. So i moved this topic to OSMC discussion.
